MANIFEST.in

```diff
@@ -1,4 +1,5 @@
-global-include *.hpp
+include pyproject.toml
```

Comment:
Old versions of `setuptools` didn't include it by default. Not sure what version they fixed it, but I included it just in case on the assumption it doesn't hurt.